packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -211,15 +215,63 @@ contract BridgePool is Testable {
     /**
      * @notice Reward relayers if a pending relay price request has a price available on the OptimisticOracle. Mark
      * the relay as complete.
-     * @dev Caller must have approved this contract to spend the deposit amount of L1 tokens to relay. There can only
-     * be one instant relayer per relay attempt and disputed relays cannot be sped up.
-     * @param _depositData Unique set of L2 deposit data that caller is trying to instantly relay.
+     * @param _depositData Unique set of L2 deposit data that caller is trying to settle a relay for.
      */
     function settleRelay(DepositData memory _depositData) public {
         bytes32 depositHash = _getDepositHash(_depositData);
         RelayData storage relay = relays[depositHash];
         require(relays[depositHash].relayState == RelayState.Pending, "Can only speed up pending slow relay");
```

Comment:
Is this require message not wrong? should it not relate to the validity of the relay action?